csrc/host_ir/pass/stream_parallel_type.h

```diff
@@ -0,0 +1,35 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <fusion.h>
+#include <preseg_passes/optimization_pass.h>
+
+namespace nvfuser::preseg_passes {
```

Comment:
```suggestion
namespace nvfuser::hir {
```

Reply:
> therefore the namespace

I don't see why a subclass needs to have the same namespace as the base class. I prefer moving the pass out of preseg_passes simply because it's not pre segmentation.

Reply:
ok nvm, let me change that then